style1.css

```diff
@@ -0,0 +1,174 @@
+body{
+    padding: 0;
+    box-sizing: border-box;
+    background-color: hsl(0, 8%, 9%);
+    margin-top: 2%;
+    
+}
+
+
+
+#container{
+    height: 90vh;
+    width: 95vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    justify-content: space-around;
+   background: linear-gradient(0deg,rgb(4, 2, 88),#550151);
+   position: relative;
+   margin: auto;
+
+}
+
+#container:before,
+#container:after{
+content: "";
+position: absolute;
+top:-7px;
+left:-7px;
+background: linear-gradient(45deg ,#fb0094,#0000ff,#00ff00,#f700ff,#eeff00,#fb0094,#0000ff,#00ff00);
+background-size:400% ;
+width: calc(100% + 14px);
+height: calc(100% + 14px);
+z-index: -1;
+animation: animate  20s linear infinite;
+}
+#container:after{
+    filter: blur(60px);
+}
+@keyframes animate{
+    0%{
+        background-color: 0 0;
+    }
+    50%{
+         background-position: 300% 0;
+    }
+    100%{
+        background-position: 0 0 ;
+    }
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+#ball1{
+   height: 200px;
+   width:200px;
+   border-radius: 50%;
+   background-color:fuchsia;
+   position: relative;
+   top: 40px;
+   animation-name: king1;
+   animation-duration: 4s;
+   animation-iteration-count: infinite;
```

Comment:
Use short hand @syedabdullahali

Reply:
Thank u for this but this was good for understand during coding